app/views/income_additional/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :header_title, 'Money & Income' %>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="step-section-header">
+      <div class="step-section-header__subhead">
+        Tell us more about your additional income.
+      </div>
+    </div>
+  </header>
+
+  <div class="form-card__content">
+    <%  income_fields = {
+      income_unemployment_insurance: "Unemployment Insurance",
+      income_ssi_or_disability: "SSI or Disability",
+      income_workers_compensation: "Worker's Compensation",
+      income_pension: "Pension",
+      income_social_security: "Social Security",
+      income_child_support: "Child Support",
+      income_foster_care: "Foster Care or Adoption Subsidies",
+      income_other: "Other Income"
+    }.stringify_keys.slice(*@additional_income) %>
+
+    <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
+
+      <% income_fields.each do |method, label| %>
+        <%= f.mb_input_field method, label, notes: ["Monthly amount"] %>
+      <% end %>
```

Comment:
My personal opinion is to explicitly put the input field tags here - which is what I did in my current branch.

On the "feels strongly about" scale, from 1 to 10 (10 being "feel VERY strongly about") I'd probably put that at a 5.

So I'll defer to you here, @jessieay. What do you think? Whichever way we land is how I'll handle this on my side too.

I do think putting the variable `income_fields` into the controller as an ivar would be a nice change though. I do feel pretty strongly that views should have view code, and not a whole lot of ruby/logic.